### PR TITLE
Updated after rebase

### PR DESCRIPTION
**Pull request by nabilwardeh**

### CODE REVIEW

File: web/src/components/views/Interest.js [hunk 1]
```diff
@@ -1,40 +1,33 @@
...
     //get the interestID from the button selected/clicked
     let interestID = evt.target.value;
-    //make the button disabled once it is selected
-    if (evt.target.disabled) {
-      evt.target.disabled = false;
+    let interests = new Set(this.state.interests);
+    if(interests.has(interestID)){
+      interests.delete(interestID);
+      this.setState({interests});
+      evt.target.className='col-sm-6 col-xs-12 col-lg-3 col-md-3 interestButton btn btn-secondary';
```
Comment (kkarimi) [inline web/src/components/views/Interest.js]: What does this do? are you trying to directly manipulate DOM? if so it's not good practice with React, it's better to just set that classname through a conditional wherever it is defined, i.e.:

```js
render() {
  const happyStyle = 'col-sm-6 col-xs-12 col-lg-3 col-md-3 interestButton btn btn-secondary';
  const sadStyle = '';
  <div className={this.state.interests ? happyClass : sadClass} />
}
```

**Comment by nabilwardeh** [comment]
This is the same code written by Amir but implementedwith a set instead of an array to make it work.

**Comment by nabilwardeh** [comment]
I am not sure how you see any direct manipulation of the dom in this. All I see is an update if the state. Please can you elaborate?

**Comment by kkarimi** [comment]
@nabilwardeh it's like 25 I am referring to `evt.target.className='...'`

**Comment by nabilwardeh** [comment]
I was writing when you commented. Right. This code was approved before and I did only change the logic to make it work. I'll all end it to change the element on render

**Pull request merged**